src/Microsoft.OData.Core/Json/ODataJsonResourceDeserializer.cs

```diff
@@ -114,6 +122,8 @@ internal void ReadResourceTypeName(IODataJsonReaderResourceState resourceState)
 
                     // Read the annotation value.
                     resourceState.Resource.TypeName = this.ReadODataTypeAnnotationValue();
+
+                 //   resourceState.TypeAnnotationFound = true;

```

Comment:
Why was this statement added then commented out? Should we remove it?

Reply:
Yep, miss to clean it.